spec/intl.html

```diff
@@ -0,0 +1,372 @@
+      </p>
+
+      <emu-alg>
+        1. <ins>If IsTemporalObject(_date_) is *true*,</ins>
+          1. <ins>Let _x_ be _date_.</ins>
+          1. <ins>If _date_ is a CivilDate object, then</ins>
+            1. <ins>Let _pattern_ be _dtf_.[[CivilDatePattern]].</ins>
+            1. <ins>Let _tm_ be { [[weekday]]: _date_.dayOfWeek, [[year]]: _date_.year, [[month]]: _date_.month, [[day]]: _date_.day }.</ins>
+          1. <ins>Else if _date_ is a CivilTime object, then</ins>
+            1. <ins>Let _pattern_ be _dtf_.[[CivilTimePattern]].</ins>
+            1. <ins>Let _tm_ be { [[hour]]: _date_.hour, [[minute]]: _date_.minute, [[second]]: _date_.second }.</ins>
+          1. <ins>Else if _date_ is a CivilDateTime object, then</ins>
+            1. <ins>Let _pattern_ be _dtf_.[[CivilDateTimePattern]].</ins>
+            1. <ins>Let _tm_ be { [[weekday]]: _date_.dayOfWeek, [[year]]: _date_.year, [[month]]: _date_.month, [[day]]: _date_.day, [[hour]]: _date_.hour, [[minute]]: _date_.minute, [[second]]: _date_.second }.</ins>
+          1. <ins>Else if _date_ is a ZonedDateTime object, then</ins>
+            1. <ins>Let _pattern_ be _dtf_.[[ZonedDateTimePattern]].</ins>
+            1. Let _offsetDateTime_ be _date_.getOffsetDateTime().
+            1. <ins>Let _tm_ be { [[weekday]]: _offsetDateTime_.dayOfWeek, [[year]]: _offsetDateTime_.year, [[month]]: _offsetDateTime_.month, [[day]]: _offsetDateTime_.day, [[hour]]: _offsetDateTime_.hour, [[minute]]: _offsetDateTime_.minute, [[second]]: _offsetDateTime_.second }.</ins>
+          1. <ins>If _pattern_ is *null*, throw a *TypeError* exception.</ins>
+        1. <ins>Else,</ins>
+          1. <ins>Let _pattern_ be _dtf_.[[Pattern]].</ins>
+          1. If _date_ is *undefined*, then
+            1. Let _x_ be Call(%Date_now%, *undefined*).
+          1. Else,
+            1. Let _x_ be ? ToNumber(_date_).
+          1. Let _x_ be TimeClip(_x_).
+          1. If _x_ is *NaN*, throw a *RangeError* exception.
+          1. <ins>Let _tm_ be ToLocalTime(_x_, _dateTimeFormat_.[[Calendar]], _dateTimeFormat_.[[TimeZone]]).</ins>
+        1. Let _locale_ be _dateTimeFormat_.[[Locale]].
+        1. Let _nfOptions_ be ObjectCreate(*null*).
+        1. Perform ! CreateDataPropertyOrThrow(_nfOptions_, `"useGrouping"`, *false*).
+        1. Let _nf_ be ? Construct(%NumberFormat%, &laquo; _locale_, _nfOptions_ &raquo;).
+        1. Let _nf2Options_ be ObjectCreate(*null*).
+        1. Perform ! CreateDataPropertyOrThrow(_nf2Options_, `"minimumIntegerDigits"`, 2).
+        1. Perform ! CreateDataPropertyOrThrow(_nf2Options_, `"useGrouping"`, *false*).
+        1. Let _nf2_ be ? Construct(%NumberFormat%, &laquo; _locale_, _nf2Options_ &raquo;).
+        1. <del>Let _tm_ be ToLocalTime(_x_, _dateTimeFormat_.[[Calendar]], _dateTimeFormat_.[[TimeZone]]).</del>
```

Comment:
It might be worth specifying in more detail how ToLocalTime works.  Right now it's very hand-wavy.

Reply:
FYI, this is being taken care of in https://github.com/tc39/ecma402/pull/348